modules/deploy/partials/kubernetes/guides/deploy-redpanda.adoc

```diff
@@ -43,10 +43,12 @@ helm repo add redpanda https://charts.redpanda.com
 helm repo update
 helm upgrade --install redpanda-controller redpanda/operator \
   --namespace <namespace> \
-  --set image.tag={latest-operator-version} \
   --create-namespace \
-  --timeout 1h
+  --timeout 1h \
+  --version {latest-operator-version} <1>
```

Comment:
Where is this value coming from? It won't matter for a little bit but this is where the difference between the tag and actual version will bite us. tag: `operator/v25.1.4` version: `v25.1-k8s4`

Reply:
We fetch the latest release of the operator during each build of the docs. This tag gets set as `latest-operator-version` in the latest version of our docs. At the moment, this version-fetching logic relies on tag names.

Will we need to build the new version format or will there be a way to get the version from the release name or something?

Reply:
Agreed in Slack that using Docker image tags would be a good solution. https://github.com/redpanda-data/docs-extensions-and-macros/pull/94